server/tests/e2e/test_api_cache.py

```diff
@@ -0,0 +1,25 @@
+import os
+import unittest
+
+import requests
+
+
+@unittest.skipIf(os.getenv("CXG_API_BASE") is None, "CXG_API_BASE not defined")
+class TestAPICache(unittest.TestCase):
```

Comment:
worth a comment this is expected to be run in a cloud env with CloudFront in place.